### ISSUE TITLE
Error in image rendering on the mobile application.

### ISSUE THREAD

**Issue by EstebanFuentealba** [flipperdevices/flipper-application-catalog#84]
When I submitted my application to the catalog (#7), I included this icon 
 [![pokemon_10px](https://github.com/flipperdevices/flipper-application-catalog/assets/442927/64409fe5-0a21-4de2-9f56-f66d1266025f)](https://github.com/EstebanFuentealba/Flipper-Zero-Game-Boy-Pokemon-Trading/blob/main/pokemon_10px.png)
, but in the mobile application, it appears like this:

![WhatsApp Image 2023-07-26 at 6 06 08 PM](https://github.com/flipperdevices/flipper-application-catalog/assets/442927/5b85af45-330f-4039-8c86-34de4cfd1303)

Perhaps it could be related to its color?

**Comment by hedger** [comment]
Yes. Images should have 1 bit color depth, as specified in the docs. That is, only allowed colors are white FFFFFF and black 000000. When the icon is being processed for the backend, all non-black pixels are transformed to transparent.

tldr: check the pixels' colors and fix them to only use pure black.

**Comment by EstebanFuentealba** [comment]
But in Flipper Zero, it appears like this.
<img width="302" alt="Captura de pantalla 2023-07-27 a la(s) 10 04 32" src="https://github.com/flipperdevices/flipper-application-catalog/assets/442927/1ba49d9e-5fbd-413f-9457-29fc6bcf9c90">

And if you say that everything that is non-black pixels are transformed to transparent, should it look like this?

![256374121-64409fe5-0a21-4de2-9f56-f66d1266025f (1)](https://github.com/flipperdevices/flipper-application-catalog/assets/442927/fb494b7b-0d9c-4cb8-a57b-147c5b6c4c04)

**Comment by hedger** [comment]
Catalog's backend uses different image processing code from FAP compilation. While FAP builder converts image to black-and-white, giving the results you see, the [specs for app icons](https://github.com/flipperdevices/flipperzero-firmware/blob/dev/documentation/AppManifests.md#parameters-for-external-applications) require 1-bit color depth. Conversion for catalog produces a transparent .png with all non-black pixels set to alpha channel.
